pkg/chartrenderer/default.go

```diff
@@ -141,14 +142,21 @@ func (r *chartRenderer) renderResources(ch *chartapi.Chart, values chartutil.Val
 		return nil, err
 	}
 
-	// Remove NODES.txt and partials
+	// Remove NOTES.txt and partials
 	for k := range files {
 		if strings.HasSuffix(k, notesFileSuffix) || strings.HasPrefix(path.Base(k), "_") {
 			delete(files, k)
 		}
 	}
 
-	manifests := manifest.SplitManifests(files)
+	manifestsMap := map[string]string{}
+	for origKey, file := range files {
+		for key, m := range releaseutil.SplitManifests(file) {
+			manifestsMap[origKey+"#"+key] = m
```

Comment:
I thinks the controllerinstallation control is using this pkg and `#` in the manifest key breaks it:

```
time="2020-02-19T10:12:49+02:00" level=info msg="Error syncing ControllerInstallation provider-gcp-5pq58: Secret \"provider-gcp-5pq58\" is invalid: [data[provider-gcp_templates_vpa.yaml#manifest-0]: Invalid value: \"provider-gcp_templates_vpa.yaml#manifest-0\": a valid config key must consist of alphanumeric characters, '-', '_' or '.' (e.g. 'key.name',  or 'KEY_NAME',  or 'key-name', regex used for validation is '[-._a-zA-Z0-9]+'),
```

Reply:
Thanks, nice catch.
I was not sure, which separator to pick...
WDYT: should we use '/' (will be replaced with '\_' in ctrlinst controller) or should we use '\_' directly?

Reply:
Both of them looks good to me.